doc/mfdata_install.md

```diff
@@ -0,0 +1,179 @@
+# Installation
```

Comment:
I think we can drop this file and keep the current installation guide

maybe we can copy it here (to be sure it is visible in the sphinx doc) in Makefile process ?

Reply:
Yes, we have to discuss how and where to make the copy (currently, the corresponding .md files are in the resources repo and not in the mfdata repo)

Reply:
the version in "resources" is only for backward compatibility

the current version is available in ".metwork-framework" subdir of mfdata